gating/pre_merge_test/run_system_tests.sh

```diff
@@ -30,14 +30,7 @@ pushd "${SYS_TEST_SOURCE}"
 git checkout "${SYS_TEST_BRANCH}"
 echo "${SYS_TEST_SOURCE} at SHA $(git rev-parse HEAD)"
 
-# Gather submodules
```

Comment:
I don't think these changes should be included. The submodule here is not one in RPC-O, but in the system tests repo.